Sprint-3/quote-generator/index.html

```diff
@@ -1,15 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
-    <script defer src="quotes.js"></script>
-  </head>
-  <body>
-    <h1>hello there</h1>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Quote Generator</title>
+  <link rel="stylesheet" href="style.css">
+  <script defer src="quotes.js"></script>
+</head>
+
+<body>
+  <div id="quote-container">
+    <h1>Hello there</h1>
     <p id="quote"></p>
     <p id="author"></p>
-    <button type="button" id="new-quote">New quote</button>
-  </body>
-</html>
+    <input type="checkbox" id="myCheckbox"> Tick me to generate a new quote<br>
```

Comment:
There should be a button which, when clicked, will show a random quote on the page. This is a required feature. 

The spec in the "stretch" section states "**Add** a toggle-switch or checkbox input to your existing quote generator". So the automatic quote generation feature should not replace original feature.

Reply:
resolved.